### ISSUE TITLE
Delete setup.py, don't keep a fake version

### ISSUE THREAD

**Issue by meerfrau** [monocongo/climate_indices#511]
**Describe the bug**
Since https://github.com/monocongo/climate_indices/commit/4a7147ee72e00dd2e4ee4d910e436575cd94787b setup.py does not properly install

**To Reproduce**
`python setup.py install --root="$pkgdir/" --optimize=1`

**Suggestion**
Delete it. It's cruel for classic self-packaging users ("Stallman's") to keep a fake version.

**Comment by monocongo** [comment]
Obviously, I'm no expert at packaging. When I moved to using `pyproject.toml` instead, the tutorial I followed advised leaving that in place. Please respond with sufficient information to persuade me that this isn't necessary. I'm easy to convince.

**Comment by monocongo** [comment]
This is fixed in the latest PR and the upcoming version 2.0.0. Enjoy!
